onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -783,7 +755,7 @@ def parse_models_info_from_directory(path, models):
 def parse_models_info_from_file(root_dir, path, models):
 
     # default working directory
-    root_working_directory = root_dir
+    root_working_directory = root_dir + 'perf/'
```

Comment:
to easily switch between docker + on machine: 
- docker perf dir lives in same workspace as trt aka /
- machine dir lives in same workspace as trt aka ~/ 
- root dir is the variable path, and perf folder holds the models